lib/web_ui/lib/src/engine/canvaskit/surface_factory.dart

```diff
@@ -46,6 +46,9 @@ class SurfaceFactory {
   /// all painting commands.
   final Surface baseSurface = Surface();
 
+  /// A surface used spcifically for `Picture.toImage` calls.
+  final Surface pictureToImageSurface = Surface();
```

Comment:
Instance fields are initialized eagerly. We might want to make it `lazy` so we only pay the initialization cost when/if the app decides to call `Picture.toImage`.